cumulus/polkadot-parachain/src/chain_spec/penpal.rs

```diff
@@ -86,7 +88,45 @@ fn penpal_testnet_genesis(
 			"safeXcmVersion": Some(SAFE_XCM_VERSION),
 		},
 		"sudo": {
-			"key": Some(Sr25519Keyring::Alice.to_account_id()),
+			"key": Some(sudo_account.clone()),
+		},
+		"assets": {
+			"assets": vec![(
+				penpal_runtime::xcm_config::TELEPORTABLE_ASSET_ID,
+				sudo_account.clone(), // owner
+				false, // is_sufficient
+				penpal_runtime::EXISTENTIAL_DEPOSIT,
+			)],
+			"metadata": vec![(
+				penpal_runtime::xcm_config::TELEPORTABLE_ASSET_ID,
+				"pal".as_bytes().to_vec(),
+				"pal".as_bytes().to_vec(),
+				12,
```

Comment:
Shouldn't Pal be what's on the balances pallet?

Reply:
Actually this is `pal-2` with asset_id set to 2, and the asset is allowed to teleport.